sample.env

```diff
@@ -110,16 +110,26 @@ FLASK_SES_NOTIFICATION_TOPICS='[]'
 
 # --- Logging
 # Logfile (will be logrotated by date)
-FLASK_LOGFILE=error.log
+FLASK_LOG_FILE=error.log
```

Comment:
Can we move this to `logs/error.log`?

There's a caveat here though.
I have already done this on the docker branch. The logs` folder has to be manually created as python logging doesn't create a folder on its own.

One solution could be to have a blank file checked in inside the `logs` folder.

Reply:
For the new server, I'm considering `/home/apps/logs/{app.name}.log` so they're all in a single folder. Also, the new default is to _not_ log to a file. I'll change it reflect here.